src/Stripe.net/Entities/Sources/SourceSourceOrderItem.cs

```diff
@@ -0,0 +1,39 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class SourceSourceOrderItem : StripeEntity<SourceSourceOrderItem>
+    {
+        /// <summary>
+        /// The amount (price) for this order item.
+        /// </summary>
+        [JsonProperty("amount")]
+        public long Amount { get; set; }
+
+        /// <summary>
+        /// This currency of this order item. Required when <c>amount</c> is
+        /// present.
+        /// </summary>
+        [JsonProperty("currency")]
+        public string Currency { get; set; }
+
+        /// <summary>
+        /// Human-readable description for this order item.
+        /// </summary>
+        [JsonProperty("description")]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// The quantity of this order item. When type is <c>sku</c>, this is
+        /// the number of instances of the SKU to be ordered.
+        /// </summary>
+        [JsonProperty("quantity")]
+        public long? Quantity { get; set; }
```

Comment:
This attribute is not flagged as nullable in the spec, so should be `long` (though maybe we can keep it nullable anyway to avoid a future breaking change... @remi-stripe wdyt?).

Reply:
I'd be fine with not nullable since `Amount` is not nullable

Reply:
Ha `Amount` *is* nullable, so we should make Quantity nullable
and also ask why one can be and not the other

Reply:
How would I figure out who to ask why quantity is not nullable in the spec?